src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -0,0 +1,393 @@
+<template>
+  <!--
+    Don't render this component until authorization object is initialized, so that we mount
+    FileUploadPreview with existing files, because its "inputFile" prop is not reactive.
+  -->
```

Comment:
Ketaki, this is one trick to using the FileUploadPreview component. (`inputFile` is only accepted when the component is rendered/mounted)